.gitignore

```diff
@@ -2,3 +2,8 @@ node_modules/
 svg/output/
 .idea/
 npm-debug.log
+svg-min/
+icon-font/
+build/
+svg-sprite/
+react/social-logo/index.jsx
```

Comment:
Updated in 9f6512c. Noticed that example.jsx is generated too, so I'm also transforming that